Week2/num4.js

```diff
@@ -0,0 +1,12 @@
+// 4
+let y = "Abdullah is doing homework";
+// 4.1
+console.log("the value of variabl y will be : Abdullah is doing the homework")
+// 4.2
+console.log(y)
+// 4.3 
+y = "I'm doing it perfectly";
+// 4.4
+console.log("the value of variable y will be : I'm doning that perfectly")
```

Comment:
Here it's the same:

"I'm **doning** **that** perfectly"

Which is not the same as the value:)